guard/tests/utils.rs

```diff
@@ -66,7 +64,7 @@ pub trait CommandTestRunner {
 
     fn run(&self, mut writer: &mut Writer, mut reader: &mut Reader) -> i32 {
         let test_app_name = String::from("cfn-guard-test");
-        let mut app = App::new(&test_app_name);
+        let mut app = clap::Command::new("cfn-guard-test");
```

Comment:
We could still use the variable here? `test_app_name`

Reply:
not sure why i changed that, good catch